ask-sdk-controls/doc/userGuide.md

```diff
@@ -288,12 +288,10 @@ launch.json file with suitable targets.
 
 ## 2.7. Get the source code
 
-To get the Controls Framework source code for perusal or for development, clone the
-ASK-SDK v2 github repo, and locate the Controls Framework project:
```

Comment:
Hi @hideokamoto ,

Thanks for the PR and being interested in our new control framework !

For these two lines, the purpose is to introduce the way how to contribute to the framework itself instead of using it. But I get your point, we also need the instruction on how to get the source code for developing skill usage.

Thus I would recommend to keep the existing two lines, but add your changes separately under this tab such as:

To get the Controls Framework source code while developer your skill, install [`ask-sdk-controls`](https://www.npmjs.com/package/ask-sdk-controls) from npm
```bash
npm install -S ask-sdk-controls
```

Reply:
For this PR, an additive change sounds good, ie Also mention how to simply get the code with `npm install -S`  sounds good.  We can tidy if necessary as we take the PR in.